src/common/components/Checkboxes/Checkboxes.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, Checkbox, useCheckboxGroup, VStack } from '@chakra-ui/react';
+import { useEffect } from 'react';
+
+import { CheckedIcon, UncheckedIcon } from '../Icons';
+
+export interface ICheckbox {
+  name: string;
+  value: string;
+  icon?: React.ReactNode;
+}
+
+interface CheckboxesProps {
```

Comment:
Could we export the interface of the component props too?

Reply:
May I know why we need to export a component props?

Reply:
Sometimes if needed we need to use the props for other components or files, but it's okay if we don't export it because right now it's not needed on the other

Reply:
Sorry my bad, I usually export this for creating a design system 😅, I'll resolve the conversation.